adserver/templates/adserver/dashboard.html

```diff
@@ -20,13 +20,13 @@ <h2>{% trans 'Staff Reports' %}</h2>
       {% trans 'Reports only visible to staff users.' %}
     </p>
     <ul>
-        <li><a href="{% url 'staff_advertisers_report' %}">{% trans 'Staff Advertiser Report' %}</a></li>
-        <li><a href="{% url 'staff_publishers_report' %}">{% trans 'Staff Publisher Report' %}</a></li>
-        <li><a href="{% url 'staff_regiontopic_report' %}">{% trans 'Staff Region & Topic Report' %}</a></li>
-        <li><a href="{% url 'staff_keyword_report' %}">{% trans 'Staff Keyword Report' %}</a></li>
-        <li><a href="{% url 'staff_geo_report' %}">{% trans 'Staff Geo Report' %}</a></li>
-        <li><a href="{% url 'staff_region_report' %}">{% trans 'Staff Region Report' %}</a></li>
-        <li><a href="{% url 'publisher_uplift_report' %}">{% trans 'Staff Uplift Report' %}</a></li>
+        <li><a href="{% url 'staff_advertisers_report' %}">{% trans 'Staff Advertiser Report' %}</a> (<a href="{% url 'staff_advertisers_report' %}?start_date={{ month_start|date:'Y-m-d' }}"> {% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'staff_publishers_report' %}">{% trans 'Staff Publisher Report' %}</a> (<a href="{% url 'staff_publishers_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'staff_regiontopic_report' %}">{% trans 'Staff Region & Topic Report' %}</a> (<a href="{% url 'staff_regiontopic_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'staff_keyword_report' %}">{% trans 'Staff Keyword Report' %}</a> (<a href="{% url 'staff_keyword_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'staff_geo_report' %}">{% trans 'Staff Geo Report' %}</a> (<a href="{% url 'staff_geo_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'staff_region_report' %}">{% trans 'Staff Region Report' %}</a> (<a href="{% url 'staff_region_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
+        <li><a href="{% url 'publisher_uplift_report' %}">{% trans 'Staff Uplift Report' %}</a> (<a href="{% url 'publisher_uplift_report' %}?start_date={{ month_start|date:'Y-m-d' }}">{% trans 'This month' %}</a>)</li>
```

Comment:
This adds a `This month` view to all the staff reports. I kept hitting issues where they were filtering by a huge date range from a previous view.